src/App.jsx

```diff
@@ -1,9 +1,9 @@
-import './App.css';
+import Footer from './components/Footer';
```

Comment:
My mistake I skipped this on the previous review, can we put the component in its own folder like src/components/Footer/<filename>

Reply:
It gives me this eslint error 
src\App.jsx
  Line 1:20:  Useless path segments for "../src/components/Footer/Footer", should be "./components/Footer/Footer"  import/no-useless-path-segments
But I add Footer folder.